tests/integration_test.py

```diff
@@ -1624,3 +1624,6 @@ def test_649(self):
         ctnr = self.client.create_container('busybox', ['sleep', '2'])
         self.client.start(ctnr)
         self.client.stop(ctnr)
+
+    def test_715(self):
```

Comment:
It's part of the TestRegressions class, which all simply reference the issue number they're testing for. But yeah, I'll add the test for the correctness of the user ID.